src/coreComponents/common/initializeEnvironment.cpp

```diff
@@ -291,16 +295,36 @@ static void addUmpireHighWaterMarks()
     // Get the total number of bytes allocated with this allocator across ranks.
     // This is a little redundant since
     std::size_t const mark = rm.getAllocator( allocatorName ).getHighWatermark();
-    std::size_t const totalMark = MpiWrapper::sum( mark );
-    std::size_t const maxMark = MpiWrapper::max( mark );
-    GEOS_LOG_RANK_0( "Umpire " << std::setw( 15 ) << allocatorName << " sum across ranks: " <<
-                     std::setw( 9 ) << LvArray::system::calculateSize( totalMark ) );
-    GEOS_LOG_RANK_0( "Umpire " << std::setw( 15 ) << allocatorName << "         rank max: " <<
-                     std::setw( 9 ) << LvArray::system::calculateSize( maxMark ) );
+
+    string minMarkValue = GEOS_FMT( "{} ({}%)",
+                                    LvArray::system::calculateSize( MpiWrapper::min( mark ) ),
+                                    ( MpiWrapper::sum( mark ) / MpiWrapper::min( mark ) ) * 100 );
+    string maxMarkValue = GEOS_FMT( "{} ({}%)",
+                                    LvArray::system::calculateSize( MpiWrapper::max( mark ) ),
+                                    ( MpiWrapper::sum( mark ) / MpiWrapper::max( mark ) )  * 100 );
+    string avgMarkValue = GEOS_FMT( "{} ({}%)",
+                                    LvArray::system::calculateSize( MpiWrapper::max( mark ) ),
+                                    ( MpiWrapper::sum( mark ) / MpiWrapper::commSize() )  * 100 );
+    string sumMarkValue = GEOS_FMT( "{} ({}%)",
+                                    LvArray::system::calculateSize( MpiWrapper::sum( mark ) ),
+                                    ( MpiWrapper::sum( mark ) / MpiWrapper::sum( mark ) ) * 100 );
+
+    tableData.addRow( allocatorName,
+                      minMarkValue,
+                      maxMarkValue,
+                      avgMarkValue,
+                      sumMarkValue );
```

Comment:
First, add the `const` keyword whenever you can.
Second, every time you call `MpiWrapper::{min, max, sum}`, you make a collective exchange. In the previous code, there was one call for each data. Please refactor your code not to multiply the call for no added value.